modules/search/Search.py

```diff
@@ -0,0 +1,80 @@
+class Search:
+    """Performs search algorithm"""
+
+    def __init__(self):
+        pass
+
+    def perform_search(self, tentGPS, planeGPS, angle):
+
+        """
+        Search program to determine the minimum rotation angle between heading and tent
+
+        Parameters
+        ----------
+        tentGPS : obj
+          dictionary containing lat and long coordinates of the tent
+        planeGPS : obj
+          dictionary containing lat and long coordinate of the plane
+        angle : float
+          bearings notation in degrees
+
+        Returns
+        -------
+        dict<"angle" : float, "counterclockwise" : bool>
+            Returns a dict with angle of rotation and whether rotation is clockwise or counterclockwise
+
+                "angle" : float
+                    Angle of rotation specified
+
+                "counterclockwise" : bool
+                    True if rotation should be counterclockwise
+                    False if rotation should be clockwise
+
+        """
+        import math
+
+        
+        '''
+            Calculating bearings between plane and tent
+        '''
+        planeLat = planeGPS["lat"]*math.pi/180
```

Comment:
Why are we converting lattitude into radians? Same with the tents.

Reply:
Latitude and Longitude are given in degrees. The math.sin, math.cos, etc. functions all require inputs to be in radians, and thus we convert our values from the start to radians.

Looking back, I could have just used the math.radians() function to convert, however both methods perform the same process.